### PR TITLE
Update sendgrid.md

### LINKED ISSUE #2457: Sendgrid SMTP port update
Re: https://pantheon.io/docs/guides/sendgrid/

## Issue Description
We want to ensure all users utilizing sendgrid have the most up to date port information compatible with the platform

## Suggested Resolution
Add new port information to sendgrid documentation

### PR DESCRIPTION
**Pull request by lauren-kelly**
Closes #2457 

## Effect
PR includes the following changes:
- update port
- update acceptable encryption options
-

## Remaining Work
- [ ] review for ambiguity on encryption options

### CODE REVIEW

**Comment by lauren-kelly** [comment]
@alexfornuto can you take a look and make sure that looks good?

File: source/_docs/guides/sendgrid.md [hunk 1]
```diff
@@ -80,8 +80,8 @@ A stable release for Drupal 8 is not yet available for the [SMTP Authentication
 4. Use the following within SMTP Server Settings:
 
  **SMTP server**: smtp.sendgrid.net <br>
- **SMTP port**: 465<br>
- **Use encrypted protocol**: Use SSL<br>
+ **SMTP port**: 2525<br>
+ **Use encrypted protocol**: Either No Encryption or TLS<br>
```
Comment (garydylina) [inline source/_docs/guides/sendgrid.md]: Is there a good reason to suggest No Encryption?

Comment (taurusismysign) [inline source/_docs/guides/sendgrid.md]: We (re)documented here what SendGrid offers on 2525 and leave for the user to decide on protocol.

Comment (alexfornuto) [inline source/_docs/guides/sendgrid.md]: We could include a parenthetical **(not recommended)**

Comment (garydylina) [inline source/_docs/guides/sendgrid.md]: I would feel better about not recommending the option we don't recommend unless we do recommend it and then our direction is clear. ;)

Comment (lauren-kelly) [inline source/_docs/guides/sendgrid.md]: @garydylina my thought on it was to head off questions of "can I just leave it as no encryption?" But seeing as we didn't see that with it saying Use SSL before, Use TLS seems fine - and we can update it later if there are problems/questions coming in

Comment (alexfornuto) [inline source/_docs/guides/sendgrid.md]: > I would feel better about not recommending the option we don't recommend unless we do recommend it and then our direction is clear. ;)

Stating the truth is not the same as recommending it. Things can go into documentation that we don't recommend, and including them does not automatically mean they are recommended.

**Comment by lauren-kelly** [comment]
@alexfornuto @garydylina  @taurusismysign Are we all set here?

**Comment by garydylina** [comment]
👍

**Pull request merged**